### DISCUSSION TITLE
[@types/auth0] Missing parameter typings for OrganizationsManager.getInvitations()

### DISCUSSION THREAD

**Discussion by pscz** [DefinitelyTyped/DefinitelyTyped#57185] [Issues with a @types package]
According to [documentation](https://auth0.github.io/node-auth0/module-management.OrganizationsManager.html#getInvitations), the type of the `params` parameter for `OrganizationsManager.getInvitations()` should also join in `CheckpointPagingOptions`, 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/03a8c197dd9c8c23ad0725f2a9e2fe37b27a4ca3/types/auth0/index.d.ts#L1372-L1389

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
see #57190

**Reply by peterblazejewicz** [reply]
merged

**Reply by pscz** [reply]
Thank you very much

**Reply by pscz** [reply]
Apparently the `management.organizations.getInvitations()` function does not actually support checkpoint pagination.
It seems the documentation was in error.

See https://github.com/auth0/node-auth0/issues/678.

Sorry for the confusion.

**Reply by peterblazejewicz** [reply]
no worries, thx for the update, reverting in #57296 I should verify this myself (though it's a complex setup involved)
